loopd/swapclient_server_test.go

```diff
@@ -1,13 +1,20 @@
 package loopd
 
 import (
+	"context"
 	"errors"
 	"testing"
 
+	"github.com/lightninglabs/lndclient"
+	"github.com/lightninglabs/loop/labels"
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+
```

Comment:
nit: don't need newline here.

this always happens and we reaaaally need a linter check for it but I'm lazy and have not done it.